GoogleDataTransport/GDTCORLibrary/GDTCOREvent.m

```diff
@@ -152,7 +167,12 @@ - (id)initWithCoder:(NSCoder *)aDecoder {
   if (self) {
     _qosTier = [aDecoder decodeIntegerForKey:qosTierKey];
     _clockSnapshot = [aDecoder decodeObjectOfClass:[GDTCORClock class] forKey:clockSnapshotKey];
-    _fileURL = [aDecoder decodeObjectOfClass:[NSURL class] forKey:fileURLKey];
+    NSURL *fileURL = [aDecoder decodeObjectOfClass:[NSURL class] forKey:fileURLKey];
+    if (fileURL) {
```

Comment:
Nice backward compatibility 👍

Reply:
+1 - it may be nice to add a comment here (in a follow up) to say why this was done for future readers.